jans-auth-server/server/pom.xml

```diff
@@ -373,23 +373,23 @@
 
 		<!-- Weld -->
 		<dependency>
-			<groupId>javax.enterprise</groupId>
-			<artifactId>cdi-api</artifactId>
+            <groupId>jakarta.validation</groupId>
+            <artifactId>jakarta.validation-api</artifactId>
 			<scope>provided</scope>
 		</dependency>
 
-        <dependency>
+       <!--  <dependency>
             <groupId>net.bootsfaces</groupId>
             <artifactId>bootsfaces</artifactId>
             <version>1.4.1</version>
             <scope>compile</scope>
         </dependency>
-
+ -->
 			<!-- OmniFaces -->
-		<dependency>
+		<!-- <dependency>
```

Comment:
We need to check if we not use this library in all xhtml (including custom authenticators). if we use it we need to use pure JSF